app/views/quotes/_quote.html.erb

```diff
@@ -0,0 +1 @@
+<div><%= quote.author %> said, <q><%= quote.body %></q>, in the <%= quote.location %></div>
```

Comment:
My logic was that we likely won't have very long quotes being uploaded to here.  Could add something to the model so that could toggle between the two depending on user preference.